<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KGncImportOptionsDlgDecl</class>
 <widget class="QWidget" name="KGncImportOptionsDlgDecl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>526</width>
    <height>679</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>526</width>
    <height>679</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>GnuCash Import Options</string>
  </property>
  <widget class="QWidget" name="layout8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>9</y>
     <width>442</width>
     <height>636</height>
    </rect>
   </property>
   <layout class="QVBoxLayout">
    <item>
     <widget class="QLabel" name="textLabel1">
      <property name="text">
       <string>Use 'Help' for more information on these options</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="buttonInvestGroupBox">
      <property name="title">
       <string>Investment Handling</string>
      </property>
      <layout class="QVBoxLayout">
       <item>
        <widget class="QRadioButton" name="radioInvest1">
         <property name="text">
          <string>One investment account for each stock</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string>buttonInvestGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioInvest2">
         <property name="text">
          <string>One investment account for all stocks</string>
         </property>
         <attribute name="buttonGroup">
          <string>buttonInvestGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioInvest3">
         <property name="text">
          <string>Prompt for an investment account for each stock</string>
         </property>
         <attribute name="buttonGroup">
          <string>buttonInvestGroup</string>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="buttonGroupBox5">
      <property name="title">
       <string>Online price quotes</string>
      </property>
      <layout class="QVBoxLayout">
       <item>
        <widget class="QCheckBox" name="checkFinanceQuote">
         <property name="text">
          <string>Use Finance::Quote for share price quotes</string>
         </property>
         <attribute name="buttonGroup">
          <string>buttonGroup5</string>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="buttonGroupBox2">
      <property name="title">
       <string>Scheduled Transactions</string>
      </property>
      <layout class="QVBoxLayout">
       <item>
        <widget class="QCheckBox" name="checkSchedules">
         <property name="text">
          <string>Drop suspect scheduled transactions</string>
         </property>
         <attribute name="buttonGroup">
          <string>buttonGroup2</string>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="buttonGroupBox4">
      <property name="title">
       <string>Decoding option</string>
      </property>
      <layout class="QVBoxLayout">
       <item>
        <layout class="QHBoxLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QCheckBox" name="checkDecode">
           <property name="text">
            <string>Decode using</string>
           </property>
           <attribute name="buttonGroup">
            <string>buttonGroup4</string>
           </attribute>
          </widget>
         </item>
         <item>
          <spacer name="spacer4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>140</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="KComboBox" name="comboDecode"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="buttonGroupBox18">
      <property name="title">
       <string>Transaction Notes option</string>
      </property>
      <widget class="QCheckBox" name="checkTxNotes">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>420</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Use transaction notes on non-split transactions</string>
       </property>
       <attribute name="buttonGroup">
        <string>buttonGroup18</string>
       </attribute>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="buttonGroupBox3">
      <property name="title">
       <string>Debug Options</string>
      </property>
      <layout class="QVBoxLayout">
       <item>
        <widget class="QCheckBox" name="checkDebugGeneral">
         <property name="text">
          <string>General debug data</string>
         </property>
         <attribute name="buttonGroup">
          <string>buttonGroup3</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkDebugXML">
         <property name="text">
          <string>Display XML data</string>
         </property>
         <attribute name="buttonGroup">
          <string>buttonGroup3</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkAnonymize">
         <property name="text">
          <string>Anonymize data</string>
         </property>
         <attribute name="buttonGroup">
          <string>buttonGroup3</string>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line1">
      <property name="frameShape">
       <enum>QFrame::HLine</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup4"/>
  <buttongroup name="buttonGroup5"/>
  <buttongroup name="buttonInvestGroup"/>
  <buttongroup name="buttonGroup2"/>
  <buttongroup name="buttonGroup18"/>
  <buttongroup name="buttonGroup3"/>
 </buttongroups>
</ui>
